core/utility/ast-utils.ts

```diff
@@ -152,26 +181,44 @@ export function addImportToNgModuleMetadata(options: AddSymbolToNgModuleOptions)
 }
 
 export function addImportToModule(options: AddImportToModuleOptions): Rule {
+  return addImportToFile({
+    filePath: options.modulePath,
+    importName: options.importName,
+    importFrom: options.importFrom
+  });
+}
+
+export function addImportToComponent(options: AddImportToComponentOptions): Rule {
```

Comment:
@umbarutiba I am not sure if we need `addImportToComponent` and `addImportToModule` wrappers as they are doing the same thing now. Can't we call `addImportToFile` instead whenever we want to add import to component or a module?

Reply:
@astorozhevsky  I also think that these wrappers are not needed.

Reply:
@umbarutiba Ok, let's move it to the single method